kubernetes/ama-logs.yaml

```diff
@@ -967,6 +967,7 @@ spec:
               - fluentdwinaks
               - "C:\\etc\\amalogswindows\\filesystemwatcher.txt"
               - "C:\\etc\\amalogswindows\\renewcertificate.txt"
+              - MonAgentCore.exe
```

Comment:
MA creates multiple processes. Is the MonAgentCore.exe main process should be monitored?

Reply:
Yes this is the main process